src/System.Globalization.Extensions/tests/IdnMapping/Data/ConformanceIdnaUnicodeTestResult.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization.Tests
+{
+    public sealed class ConformanceIdnaUnicodeTestResult : ConformanceIdnaTestResult
+    {
+        public bool ValidDomainName { get; private set; }
+
+        public ConformanceIdnaUnicodeTestResult(string entry, string fallbackValue, bool validDomainName = true)
+            : base(entry, fallbackValue)
+        {
+            ValidDomainName = validDomainName;
+        }
+    }
+}
```

Comment:
Do we really need to have this new type? can't we just combine this with the ConformanceIdnaTestResult? either you'll have ConformanceIdnaASCIITestResult too or just keep one type covering both. 

I am ok you keep your change here if you feel strongly about it.